spec/models/observation_spec.rb

```diff
@@ -1511,49 +1508,53 @@
       end
     end
 
-    it "should increment the taxon's counter cache" do
-      o = Observation.make!
-      t = Taxon.make!
-      expect(t.observations_count).to eq(0)
-      o.update_attributes(:taxon => t)
-      Delayed::Job.find_each{|j| j.invoke_job}
-      t.reload
-      expect(t.observations_count).to eq(1)
-    end
-  
-    it "should increment the taxon's ancestors' counter caches" do
-      o = Observation.make!
-      p = without_delay { Taxon.make!(rank: Taxon::GENUS) }
-      t = without_delay { Taxon.make!(parent: p, rank: Taxon::SPECIES) }
-      expect(p.observations_count).to eq 0
-      o.update_attributes(:taxon => t)
-      Delayed::Job.find_each{|j| j.invoke_job}
-      p.reload
-      expect(p.observations_count).to eq 1
-      Observation.elastic_index!(ids: [ o.id ], delay: true)
-      p.reload
-      expect(p.observations_count).to eq 1
-    end
+    describe "counter caches updating" do
+      with_es_enabled_for_group(Observation, Taxon)
```

Comment:
Didn't change the specs here, just indented them inside `describe` block to enable ElasticSearch only for these examples.